server/container/cli/src/test/java/org/apache/james/cli/ServerCmdTest.java

```diff
@@ -865,25 +729,21 @@ public void copyMailboxMappingCommandShouldThrowOnMissingArguments() throws Exce
         String[] arguments = { "-h", "127.0.0.1", "-p", "9999", CmdType.COPYMAILBOX.getCommand(), srcBean};
         CommandLine commandLine = ServerCmd.parseCommandLine(arguments);
 
-        control.replay();
         try {
             testee.executeCommandLine(commandLine);
         } finally {
-            control.verify();
-        }
+            }
     }
 
     @Test(expected = InvalidArgumentNumberException.class)
     public void deleteUserMailboxesMappingCommandShouldThrowOnMissingArguments() throws Exception {
         String[] arguments = { "-h", "127.0.0.1", "-p", "9999", CmdType.DELETEUSERMAILBOXES.getCommand()};
         CommandLine commandLine = ServerCmd.parseCommandLine(arguments);
 
-        control.replay();
         try {
             testee.executeCommandLine(commandLine);
         } finally {
-            control.verify();
```

Comment:
you dropped the verify stage ?

Reply:
in some tests there is no expectation, and I think I can remove the `control.verify()` which is used to verify expectations.